spec/requests/api/people_controller_index_spec.rb

```diff
@@ -55,14 +55,22 @@
 
       before do
         allow(People::Finder).to receive(:new).and_return(people_finder)
-        allow(Moves::ImportPeople).to receive(:new).with([prison_number])
+        allow(Moves::ImportPeople).to receive(:new).with([prison_number.upcase])
```

Comment:
We implicitly test that a prison number is always received in it's upcased form.